src/components/candidateContent/candidateInfo/CandidateInfo.jsx

```diff
@@ -27,25 +36,82 @@ const CandidateInfo = (props) => {
     return newInfo;
   };
 
+  const onButtonCancel = () => {
+    setIsEditModeOn(false);
+  };
+
+  const handleChange = (index) => (e) => {
+    console.log(`Best => ${e.target.value}`);
+    const newArr = { ...newValues };
+    newArr[index] = e.target.value;
+    setNewValues(newArr);
+  };
+
   const formatedInfo = formatInfo(candidateInfo);
 
+  const onButtonEdit = () => {
+    setNewValues({ ...formatedInfo });
+    setIsEditModeOn(true);
+  };
+
+  const onButtonSave = () => {
+    console.log(`SAVE`);
+
+    newValues.isPlanningToJoin = true;
+
+    newValues.registrationDate = dayjs.utc(
+      newValues.registrationDate,
+      'DD/MM/YYYYTHH:mm');
+    newValues.bestContactTime = dayjs.utc(newValues.bestContactTime, 'HH:mm');
+
+    dispatch(updateCandidateInfo(newValues));
+    setIsEditModeOn(false);
+  };
+
   return (
     <Box>
       <Box display="flex" flexDirection="row" justifyContent="space-between">
-        <Typography paddingLeft="1%" variant="h4">
-          {formatedInfo.fullName}
-        </Typography>
-        <Button variant="outlined">{getFieldLabel('common.edit')}</Button>
+        {!isEditModeOn && (
+          <Typography paddingLeft="1%" variant="h4">
+            {`${formatedInfo.firstName} ${formatedInfo.lastName}`}
+          </Typography>
+        )}
+        {isEditModeOn && (
+          <Input
```

Comment:
Better to have two separate inputs for first name and last name

Reply:
Will be after setting edit mode to the form